includes/classes/class-omise-charges-table.php

```diff
@@ -2,113 +2,114 @@
 defined( 'ABSPATH' ) or die( "No direct script access allowed." );
 
 if ( ! class_exists( 'WP_List_Table' ) ) {
-    require_once( ABSPATH . 'wp-admin/includes/class-wp-list-table.php' );
+	require_once( ABSPATH . 'wp-admin/includes/class-wp-list-table.php' );
 }
 
 if ( ! class_exists( 'Omise_Charges_Table' ) ) {
-    class Omise_Charges_Table extends WP_List_Table {
-
-        function __construct( $omise_list_object ) {            
-            parent::__construct( array(
-                'singular' => 'omise_charge',
-                'plural'   => 'omise_charges',
-                'ajax'     => false
-            ) );
-
-            $this->items = $omise_list_object;
-        }
-
-        function get_table_classes() {
-            return array(
-                'widefat',
-                'striped',
-                $this->_args['plural']
-            );
-        }
-
-        function prepare_items() {
-            $columns    = $this->get_columns();
-            $hidden     = array();
-            $sortable   = array();
-
-            $totalitems = $this->items['total'];
-            $perpage    = $this->items['limit'];
-            $paged      = isset( $_GET['paged'] ) ? $_GET['paged'] : 1;
-            $totalpages = ceil( $totalitems / $perpage );
-            
-            $this->set_pagination_args( array(
-                'total_items' => $totalitems,
-                'total_pages' => $totalpages,
-                'per_page'    => $perpage
-            ) );
-
-            $this->_column_headers = array(
-                $columns,
-                $hidden,
-                $sortable
-            );
-
-            // Clear item property to call `no_items` method.
-            if ( $this->items['total'] <= 0 ) {
-                $this->items = array();
-            }
-        }
-
-        function get_columns() {
-            return $columns = array(
-                'chrg_amount'     => __( 'Amount' ),
-                'chrg_id'         => __( 'Charge Id' ),
-                'chrg_authorized' => __( 'Authorized' ),
-                'chrg_paid'       => __( 'Paid' ),
-                'chrg_failure'    => __( 'Failure Message' ),
-                'chrg_datetime'   => __( 'Created' ),
-                'chrg_action'     => '',
```

Comment:
@nimid can you make/upload screenshot of this changed to PR's desc (that you removed `chrg_action` column).

thanks